receiver/sqlserverreceiver/documentation.md

```diff
@@ -256,6 +256,22 @@ This metric is only available when the receiver is configured to directly connec
 | ---- | ----------- | ---------- |
 | {processes} | Gauge | Int |
 
+### sqlserver.properties.db.status
```

Comment:
The name is confusing. `db.status` defines an attribute; it shouldn't be part of a metrics name. Can we call it `sqlserver.database.count`?

Reply:
That's fair, I can understand it would be confusing. `count` makes sense 👍

Reply:
Done.